### PR TITLE
Add Linptech Human Presence Sensing Flat Panel Light

### PR DESCRIPTION
**Pull request by xiasi0**
Add Linptech Human Presence Sensing Flat Panel Light: linp.light.lp1bc
https://home.miot-spec.com/spec/linp.light.lp1bc
image

### CODE REVIEW

File: custom_components/xiaomi_gateway3/core/devices.py [hunk 1]
```diff
@@ -3130,6 +3130,18 @@
...
     ]
+}, {
+    # https://home.miot-spec.com/spec/linp.light.lp1bc
+    19653: ["Linptech", "Human Presence-Sensing Flat Panel Light", "LP1", "light.lp1bc"],
+    "spec": [
+        BaseConv("light", "light", mi="2.p.1"),
+        BrightnessConv("brightness", mi="2.p.2", max=100),
+        MapConv("mode", "select", mi="2.p.7", map={0: "None", 1: "Reading", 3: " Lighting", 4: "Night light"}),
+        MapConv("power_on_state", "select", mi="2.p.9", map={0: "default", 1: "on", 2: "off"}),
+        MapConv("occupancy_status", "sensor", mi="5.p.1", map={0: "NoOne", 1: "SomeOne"}),
```
Comment (Necroneco) [inline custom_components/xiaomi_gateway3/core/devices.py:3141]: `BoolConv("occupancy", "binary_sensor", mi="5.p.1"),` might be better

Comment (xiasi0) [reply custom_components/xiaomi_gateway3/core/devices.py]: Change occupation to boolean type

image

**Comment by AlexxIT** [comment]
You have wrong model. It should be `linp.light.lp1bc`

**Comment by AlexxIT** [comment]
Thanks!

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/XiaomiGateway3/releases/tag/v4.0.7

**Pull request merged**